en/sim_gazebo_gz/vehicles.md

```diff
@@ -41,6 +41,14 @@ make px4_sitl gz_x500_depth
 
 ![x500 with depth camera in Gazebo](../../assets/simulation/gazebo/vehicles/x500_depth.png)
 
+### X500 Quadrotor with Monocular Camera
+
+This models has a simple monocular camera sensor attached (there is no physical camera visualization on the model itself).
```

Comment:
So essentially a model you can use to stream video from the model into QGC?

Reply:
@hamishwillee This is not yet available for QGC. Camera manager and GST plugin that were available in Gazebo-classic are not yet present here. I was working on this about a month ago but got sidetracked. I am looking to get back to it when I find the time.

Reply:
Cool. I've merged the note and will now merge this. Thanks for your patience!